Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Services/LocatorStorageService.cs

```diff
@@ -12,12 +13,24 @@ public class LocatorStorageService
     //TODO: Impl factory
     public T Get<T>(string elementName)
     {
-        //TODO:impl search service, check type is exist
-        var mainPage = typeof(MainPage);
-        var page = Activator.CreateInstance(mainPage);
+        var type = typeof(ISelectorStorage);
+        var types = AppDomain.CurrentDomain.GetAssemblies()
+            .SelectMany(s => s.GetTypes())
+            .Where(p => type.IsAssignableFrom(p) && p.IsClass);
 
-        var element = (T)mainPage.GetField(elementName.ToCamelCase())?.GetValue(page)!;
-        if (element is null) throw new Exception($"'{elementName}' selectors not found.");
-        return element;
+        IWebElement element = null;
+        foreach (var pageType in types)
+        {
+            var pageTemp = Activator.CreateInstance(pageType);
+            var temp = (IWebElement) pageType.GetField(elementName.ToCamelCase())?.GetValue(pageTemp)!;
```

Comment:
`pageTemp` and `temp` variable names are not very clear, is there any other names you can come up with?

Reply:
Changed pageTemp to instanceOfClassWithElements
Changed temp to classField

Added checking for empty element name:
        if (elementName.Equals("")) throw new Exception("element name can not be empty. Please correct test step");